.github/workflows/ci.yml

```diff
@@ -93,3 +94,62 @@ jobs:
         env:
           CI_NAME: github-actions
           COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+
+  publish:
+    if: >-
+        ((github.event_name == 'workflow_dispatch') || (github.event_name == 'push')) &&
+        (github.ref == 'refs/heads/pakrym/release-pipeline' || startsWith(github.ref, 'refs/tags/v')) &&
```

Comment:
Are you still testing this on your branch? If so let's skip the tag push for now and only do this on pushes to your branch?

Reply:
Removed the tag trigger.